.konflux/catalog/fbc-images-resolvable-integration-test-idms.yaml

```diff
@@ -0,0 +1,13 @@
+# We use this IDMS to check if the images in the FBC are resolvable using the staging or production catalog.
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageDigestMirrorSet
+metadata:
+  name: fbc-images-resolvable-integration-test-idms
+spec:
+  imageDigestMirrors:
+  - mirrors:
+    - registry.stage.redhat.io/openshift4/numaresources-rhel9-operator
+    source: registry.redhat.io/openshift4/numaresources-rhel9-operator
+  - mirrors:
+    - registry.stage.redhat.io/openshift4/numaresources-operator-bundle
+    source: registry.redhat.io/openshift4/numaresources-operator-bundle
```

Comment:
when backporting this make sure you update the rhel9 reference to rhel8 at the appropriate z stream, I forgot to do that